components/calculation/MeasurePopulations.tsx

```diff
@@ -0,0 +1,45 @@
+import { Enums } from 'fqm-execution';
```

Comment:
This file isn't really a component and should probably be just included in the PopulationComparisonTable file or moved to the `utils` folder.